frontends/infinite-corridor/src/scss/learning-resources.scss

```diff
@@ -1,17 +1,16 @@
 @use "theme";
+@use "breakpoints";
 @use "ol-search-ui/assets/learning-resource-card" with (
   $lightText: theme.$font-color-light
 );
 
 @use "ol-search-ui/assets/learning-resource-drawer";
 
-@import "./breakpoints";
-
 $cardListSpacer: 1.5em;
 
 .ic-resource-card {
 
-  @include breakpoint(phone) {
+  @include breakpoints.down(sm) {
```

Comment:
This is slightly different (old: 580px, new: 600px) but the general behavior is unchanged